.gitignore

```diff
@@ -2,3 +2,5 @@ stulto
 pkg
 *.pkg.tar.*
 .idea/
+
+/build
```

Comment:
https://mesonbuild.com/FAQ.html#how-do-i-ignore-the-build-directory-in-my-vcs

You don't need this, meson sets it up automatically for you. :)

Reply:
Way late because I missed the GH notifications, thanks for the catch!

Reply:
No problem, always happy to help. :)